packages/xstate-fsm/test/fsm.test.ts

```diff
@@ -140,6 +146,14 @@ describe('@xstate/fsm', () => {
     }).toThrow();
   });
 
+  it('should throw an error for undefined next state config', () => {
+    expect(() => {
+      lightFSM.transition('green', 'TARGET_INVALID');
```

Comment:
could you create a separate machine for this test? we try to not reuse machines between tests right now - the rest of the tests has been written when there was no such decision yet and they just haven't been rewritten yet

Reply:
Sure, I'd be happy to! Shouldn't that be in a separate PR though?

Reply:
It's a new test - not asking about refactoring the old one, just using a separate machine for this new one.

Reply:
Can even have a machine like this:

```js
const testMachine = createMachine({
  initial: 'green',
  states: {
    green: {
      on: { TARGET_INVALID: 'blue' }
    },
    yellow: {}
  }
});
```

If you want to copy-paste the above.

Reply:
Thanks! I just pushed the changes.